packages/next/server/config-schema.ts

```diff
@@ -384,6 +384,9 @@ const configSchema = {
           },
           type: 'array',
         },
+        useUndici: {
```

Comment:
Nit pick: this config should probably be `enableUndici` since "use" is typically a react hook

Reply:
I'm happy with this change @balazsorban44 do you have any preference?

Reply:
Sounds good to me too. Let's change it :+1:

Another option: `fetchPolyfill: "undici"`?

Reply:
I think that implies we support other fetch polyfills. maybe we can in the future. but for now I think the goal is strictly lets start using undici which will eventually become the global fetch